Sources/FTPropertyWrappers/Serialized.swift

```diff
@@ -32,7 +45,8 @@ public final class Serialized<Value> {
         }
     }
 
-    /// It is enouraged to use this method to make more complex operations with the stored property, like read-and-write. Do not perform any time-demading operations in this block since it will stop other uses of the stored property.
+    /// This method dispatches it's block argument on the synchronization queue, allowing user to modify enclosed value. It is enouraged to use this method to make more complex operations with the stored property, like read-and-write. Do not perform any time-demading operations in this block since it will stop other uses of the stored property.
```

Comment:
```suggestion
    /// This method dispatches its block argument on the synchronization queue, allowing user to modify enclosed value. It is enouraged to use this method to make more complex operations with the stored property, like read-and-write. Do not perform any time-demading operations in this block since it will stop other uses of the stored property.
```